Containerfile.common

```diff
@@ -37,7 +37,7 @@ RUN /tmp/build-kmod-v4l2loopback.sh
 RUN /tmp/build-kmod-wl.sh
 
 # Exclude negativo17 kmods from Fedora 39
-RUN if grep -q ${FEDORA_MAJOR_VERSION}; then \
+RUN if grep -qv "39" <<< ${FEDORA_MAJOR_VERSION}; then \
```

Comment:
LGTM - but I don't know grep...  

More like looks good to GH Copilot :)

Reply:
We use the same implementation over in Bazzite for checking image details :). Should be good!